src/llvm/instruction_collection.cr

```diff
@@ -18,4 +18,12 @@ struct LLVM::InstructionCollection
   private def llvm_first
     LibLLVM.get_first_instruction @basic_block
   end
+
+  def each : Nil
```

Comment:
ditto re: `include Enumerable`

Reply:
Once it's there you can remove `first?`/`first` methods, since they're included along with `Enumerable` module.